src/NeoSharp.Core/Wallet/NEP6/NEP6WalletManager.cs

```diff
@@ -272,10 +274,22 @@ private byte[] GetPrivateKeyFromWIF(string wif)
             }
         }
 
-
+        /// <summary>
+        /// save the open wallet into the same file
+        /// </summary>
         public void SaveWallet()
         {
-            //throw new NotImplementedException();
+            SaveWallet(_openWalletFilename);
```

Comment:
check null ?

Reply:
Ok, I have checked on the other SaveWallet, so I think we are safe now